packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -111,7 +111,7 @@ class TabController extends ChangeNotifier {
     }
   }
 
-  Null _indexChanged(dynamic value) {
+  void _indexChanged() {
```

Comment:
you can just inline this function now. it was only here because i couldn't inline it before.